best_practices/business-logic.rst

```diff
@@ -198,7 +198,7 @@ looking for mapping information::
 
         /**
          * @ORM\OneToMany(
-         *      targetEntity="Comment",
+         *      targetEntity="App\Entity\Comment",
```

Comment:
I think it is better to suggest `Comment::class` because if you refactor namespace it will be picked up and you don't have to edit a bunch of strings

Reply:
The similar approach was applied before, so I decide to follow it. Do you want to suggest rework everywhere to `ClassNmae::class`?

Reply:
Ah, then leave it like this in PR. I think reworking it everywhere would be a nice addition. Not sure what docs team thinks about it

Reply:
If team decide to rework everything, so, I'm in ;)

Reply:
It works, I used it like:

```
     * @ORM\ManyToOne(targetEntity=Comment::class)
     * @ORM\JoinColumn(name="id", referencedColumnName="id")
```

When you refactor and maybe change the namespace, it changes the use statements (not sure about strings but it didn't work for me most of the times) so that is an advantage on using `Comment::class` instead of `"App\Entity\Comment"`.

Reply:
After discussing this with @ostrolucky on slack, it seems that\=

> clicking on `mappedBy="foo"`/`inversedBy="foo"` 

dosen't work with `Comment::class` also if you rename the class it still stays `Comment::class` so I guess FQCN is still better

Reply:
You mean "clicking on" IDE?

Reply:
ye, clicking on the name of the var in your IDE (phpstorm in my case).